scripts/docker-build

```diff
@@ -88,7 +97,7 @@ pushd "${tmpdir}" > /dev/null
 
   echo "Building docker image '${image_name}'..."
   cp "${release_dir}/docker-prod/Dockerfile" .
-  docker build -t "${image_name}" .
+  docker build --build-arg TARGETARCH=${target_arch} -t "${image_name}" .
```

Comment:
Only add `--build-arg TARGETARCH=${target_arch}` if `${target_arch}` is non-empty